src/layers/legacy/medVtkInria/vtkImageView/vtkImage3DDisplay.h

```diff
@@ -38,15 +51,14 @@ class vtkImage3DDisplay : public vtkObject
     vtkSetMacro(ColorLevel, double);
     vtkGetMacro(ColorLevel, double);
 
-    vtkAlgorithmOutput* GetOutputPort();
+    virtual vtkImageAlgorithm* GetInputProducer() const { return this->InputProducer; }
```

Comment:
i would prefer even small functions in the cpp, because it's easier to follow the processes in the cpp than in the 2 files, but it's ok

Reply:
Ok